wewatch-client/src/views/swiping/swiping-container.js

```diff
@@ -97,7 +106,10 @@ const SwipingContainer = (props) => {
       </Col>
```

Comment:
Maybe the conditional block to render the swiping completion screen could start at line 81 (i.e. if !swipingCompleted, render <Col>...movie details...</Col>, else render SwipingCompletedScreen), and we could add the SwipingCompletedScreen render here